Sources/Rules/EmptyExtension.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  EmptyExtension.swift
+//  SwiftFormat
+//
+// Created by manny_lopez on 7/29/24.
+//  Copyright © 2024 Nick Lockwood. All rights reserved.
+//
+
+import Foundation
+
+public extension FormatRule {
+    /// Remove empty, non-conforming, extensions
+    static let emptyExtension = FormatRule(help: "Remove empty, non-conforming, extensions") { formatter in
```

Comment:
Maybe we should have this rule always run after the `unusedPrivateDeclaration` rule

```suggestion
    static let emptyExtension = FormatRule(
        help: "Remove empty, non-conforming, extensions",
        orderAfter: [.unusedPrivateDeclaration]
    ) { formatter in
```

You can also write a test case that these two rules work together as expected, for example:

```swift
func testRemoveUnusedPrivateDeclarationThenEmptyExtension() {
    let input = """
    class Foo {
        init() {}
    }

    extension Foo {
        private var bar: Bar { "bar" }
    }
    """

    let output = """
    class Foo {
        init() {}
    }

    """

    testFormatting(for: input, [output], rules: [.unusedPrivateDeclaration, .emptyExtension])
}  
```

Reply:
Added!